dds_cli/__main__.py

```diff
@@ -1033,134 +828,323 @@ def abort(click_ctx, username, project):
         sys.exit(1)
 
 
-# COMMAND##########################################################################
-#### USER ################################################################## USER #
-###################################################################################
-@dds_main.group(no_args_is_help=True)
+####################################################################################################
+####################################################################################################
+## DATA #################################################################################### DATA ##
+####################################################################################################
+####################################################################################################
+
+
+@dds_main.group(name="data", invoke_without_command=True)
 @click.pass_obj
-def user(click_ctx):
-    """Manage user accounts, including your own."""
-    pass
-
-
-# SUBCOMMAND#######################################################################
-####### ADD ################################################################# ADD #
-###################################################################################
-## Invite new users to the system
-@user.command()
-@click.argument(
-    "email",
-    nargs=1,
-    type=str,
-    required=True,
+def data_group_command(_):
+    """Group command: dds project. Manage projects."""
```

Comment:
`dds project. Manage projects` maybe a type since the command is `dds data `here?

Reply:
Fixed!